pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -306,15 +311,71 @@ public static void validateIngestionConfig(TableConfig tableConfig, @Nullable Sc
         }
       }
 
+      // Aggregation configs
+      List<AggregationConfig> aggregationConfigs = ingestionConfig.getAggregationConfigs();
+      Set<String> aggregationSourceColumns = new HashSet<>();
+      if (aggregationConfigs != null) {
```

Comment:
To be consistent with the check in mutable segment
```suggestion
      if (!CollectionUtils.isEmpty(aggregationConfigs)) {
```

Reply:
done